collector/cluster_health.go

```diff
@@ -217,21 +237,31 @@ func (c *ClusterHealth) fetchAndDecodeClusterHealth() (clusterHealthResponse, er
 	}
 
 	if err := json.NewDecoder(res.Body).Decode(&chr); err != nil {
+		c.jsonParseFailures.Inc()
 		return chr, err
 	}
 
 	return chr, nil
 }
 
 func (c *ClusterHealth) Collect(ch chan<- prometheus.Metric) {
+	c.totalScrapes.Inc()
+	defer func() {
+		ch <- c.up
+		ch <- c.totalScrapes
+		ch <- c.jsonParseFailures
+	}()
+
 	clusterHealthResponse, err := c.fetchAndDecodeClusterHealth()
 	if err != nil {
+		c.up.Set(0)
```

Comment:
Oh, I see ... this slightly changed after rebasing onto the current master.

What's your suggestion here?